webapp/src/main/java/com/box/l10n/mojito/rest/ai/AIPromptWS.java

```diff
@@ -43,21 +51,30 @@ public void addPromptToRepository(
     promptService.addPromptToRepository(promptId, repositoryName, promptType);
   }
 
+  @Operation(summary = "Delete a Prompt by ID")
   @RequestMapping(value = "/api/ai/prompts/{prompt_id}", method = RequestMethod.DELETE)
   @Timed("AIWS.deletePrompt")
   public void deletePrompt(@PathVariable("prompt_id") Long promptId) {
     logger.debug("Received request to delete prompt id {}", promptId);
     promptService.deletePrompt(promptId);
   }
 
-  @RequestMapping(value = "/api/ai/prompts/{prompt_id}", method = RequestMethod.GET)
+  @Operation(summary = "Get a Prompt by ID")
+  @RequestMapping(
+      value = "/api/ai/prompts/{prompt_id}",
+      method = RequestMethod.GET,
+      produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
Is adding `produces` and `consumes` to the `RequestMapping` annotation a requirement for the Swagger documentation generation?

If not it might be easier to exclude the addition of these additions from this PR, then in a later PR you can add them in after testing against the relevant endpoint with the mojito-cli. Concern is that they will cause issues as seen previously that might not be caught in tests.

Reply:
Done